dataclass_wizard/serial_json.py

```diff
@@ -10,6 +8,8 @@
 from .dumpers import asdict
 from .loaders import fromdict, fromlist
 from .type_def import Decoder, Encoder, JSONObject, ListOfJSONObject
+# noinspection PyProtectedMember
+from .utils.compatibility import _create_fn, _set_new_attribute
```

Comment:
I am wondering if we can come up with a better name for the module. Maybe something like `dataclasses_compat`? I think `compatability` is a bit vague overall.